definitions-taxonomy/Product.md

```diff
@@ -0,0 +1,3 @@
+# Product
+
+Product is any deliverable (e.g. software, hardware, specification, services, etc.) which can be referred to with a name. This applies regardless of the origin, the license model, or the mode of distribution of the deliverable.
```

Comment:
Suggest to leave out one of the open end specifiers like so:

Instead of:
>(e.g. software, hardware, specification, services, etc.)

Write:
>(e.g. software, hardware, specification, or services)

Reply:
I guess the idea was to strengthen that those examples are not a closed list... (But I'm okay with it either way.)

Reply:
Both "e.g." (abbreviation for the Latin phrase exempli gratia, which means “for example) and "etc." on their own accomplish to state a list that may have more members than stated.